pkg/constants/constants.go

```diff
@@ -6,6 +6,8 @@ import (
 	"time"
 )
 
+type Visibility bool
```

Comment:
pls change the type name to something like HostRemoteAccess or something like that

Reply:
changed to PublicHTTPAccess